src/omnisharp/server.ts

```diff
@@ -286,11 +342,24 @@ export class OmniSharpServer {
         });
     }
 
-    public stop(): Promise<void> {
-
-        while (this._disposables.length) {
-            this._disposables.pop().dispose();
+    // we could move this to a project updated observer and pass the server there. But we will need the event stream also there :(
```

Comment:
I don't understand this comment, but I agree with the `:(`

Reply:
Sorry I had added in while working but forgot to delete :(